### PR TITLE
Update Glorious

### PR DESCRIPTION
**Pull request by phallobst**

### CODE REVIEW

**Review by hkamran80 — APPROVED** [review]

File: entries/g/gloriousgaming.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Glorious": {
+    "domain": "gloriousgaming.com",
+    "contact": {
+      "facebook": "GloriousPCG",
```
Comment (hkamran80) [inline entries/g/gloriousgaming.com.json]:
```suggestion
```

The check is failing

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/g/gloriousgaming.com.json (hunk 1 above)
Comment (Carlgo11) [reply entries/g/gloriousgaming.com.json]: Looks like Facebook is restricting access to some pages more than others.
We might need a Facebook API key in the future :unamused: 
In the meantime I second @hkamran80's suggestion ^

Comment (johntheholman) [reply entries/g/gloriousgaming.com.json]: The page exists though and you can access it (like some of the `domain` changes I have been working on lately). Is there something that may need to be updated with the tests?

Screen Shot 2023-02-02 at 2 55 22 AM

Comment (hkamran80) [reply entries/g/gloriousgaming.com.json]: Like @Carlgo11 said, we might need to add an API key for Facebook. This removal is temporary, at least until we can fix this bug.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**